database/seeds/CentreUsersSeeder.php

```diff
@@ -50,7 +50,7 @@ public function run()
             "password" => bcrypt('store_pass'),
             "role" => "foodmatters_user",
         ]);
-        $user3->centres()->attach(1, ['homeCentre' => true]);
+        $user3->centres()->attach(2, ['homeCentre' => true]);
```

Comment:
I'm guessing there is a good reason to go for second centre.

Reply:
Yes, the second centre gets a different print_pref, and at least one user ought to have it so karl cna test.